python/lsst/daf/butler/_butler.py

```diff
@@ -1229,7 +1230,13 @@ def put(
 
         return ref
 
-    def getDirect(self, ref: DatasetRef, *, parameters: Optional[Dict[str, Any]] = None) -> Any:
+    def getDirect(
+        self,
+        ref: DatasetRef,
+        *,
+        parameters: Optional[Dict[str, Any]] = None,
+        readStorageClass: Optional[Union[StorageClass, str]] = None,
```

Comment:
As a general question, if this override facility is going to be used a lot, should we consider a shorter parameter name to make it easier to use? `read_sc`?

cc/ @erykoff